pages/custom-query-modules/cpp/cpp-api.md

```diff
@@ -22,7 +22,7 @@ Memgraph in order to use them.
 With this API it’s possible to extend your Cypher queries with **functions** and **procedures** with
 `AddProcedure` and `AddFunction`.
 
-The API needs memory access to add procedures and functions, this can be done with either `mgp::MemoryDispatcherGuard guard(memory);` or `mgp::memory = memory;`, where the use of the former is advised since `mgp::memory = memory;` is not thread-safe and will be deprecated in the future.
+The API needs memory access to add procedures and functions, this can be done with `mgp::MemoryDispatcherGuard guard(memory);`. Old code used `mgp::memory = memory;`, but this is not thread-safe and has been deprecated. v2.18.1 onwards you should modify your C++ modules and recompile.
```

Comment:
Should or must?

Reply:
v2.18.1 still allows this, but the user should be careful and not execute procedures that share the memory in parallel.
v2.21 setting mgp::memory will cause a compilation error.

Reply:
Then please change the language in this PR.

Reply:
Added the required note about v2.21.